middleware/bottle.js

```diff
@@ -0,0 +1,34 @@
+import Bottleneck from "bottleneck";
```

Comment:
did we end up actually using Bottleneck? If not, we can go ahead and delete this whole thing.